.cirrus.yml

```diff
@@ -39,742 +28,97 @@ env:
-
-    timeout_in: 40m
-
-    setup_environment_script: '$SCRIPT_BASE/setup_environment.sh |& ${TIMESTAMP}'
-    integration_test_script: '$SCRIPT_BASE/integration_test.sh |& ${TIMESTAMP} | ${LOGFORMAT} integration_test'
-
-    on_failure:
-        failed_branch_script: '$CIRRUS_WORKING_DIR/$SCRIPT_BASE/notice_branch_failure.sh'
-
-    always:
-        <<: *standardlogs
-
-
-special_testing_endpoint_task:
-
-    depends_on:
-        - "gating"
-        - "varlink_api"
-        - "vendor"
-
-    only_if: >-
-        $CIRRUS_CHANGE_MESSAGE !=~ '.*CI:IMG.*' &&
-        $CIRRUS_CHANGE_MESSAGE !=~ '.*CI:DOCS.*'
-
-    env:
-        SPECIALMODE: 'endpoint'  # See docs
-
-    timeout_in: 20m
-
-    setup_environment_script: '$SCRIPT_BASE/setup_environment.sh |& ${TIMESTAMP}'
-    integration_test_script: '$SCRIPT_BASE/integration_test.sh |& ${TIMESTAMP} | ${LOGFORMAT} integration_test'
-
-    on_failure:
-        failed_branch_script: '$CIRRUS_WORKING_DIR/$SCRIPT_BASE/notice_branch_failure.sh'
-
-    always:
-        <<: *standardlogs
-
-
-# Test building of new cache-images for future PR testing, in this PR.
-test_build_cache_images_task:
-
-    only_if: >-
-        $CIRRUS_BRANCH != $DEST_BRANCH &&
-        $CIRRUS_CHANGE_MESSAGE =~ '.*CI:IMG.*' &&
-        $CIRRUS_CHANGE_MESSAGE !=~ '.*CI:DOCS.*'
+              TARGET_NAME: $PRIOR_FEDORA_NAME
+              VM_IMAGE_NAME: $PRIOR_FEDORA_CACHE_IMAGE_NAME
+        - name: 'Build for $UBUNTU_NAME'
+          env:
+              TARGET_NAME: $UBUNTU_NAME
+              VM_IMAGE_NAME: $UBUNTU_CACHE_IMAGE_NAME
+        - name: 'Build for $PRIOR_UBUNTU_NAME'
+          env:
+              TARGET_NAME: $PRIOR_UBUNTU_NAME
+              VM_IMAGE_NAME: $PRIOR_UBUNTU_CACHE_IMAGE_NAME
+    # script: echo "Stub Build for $VM_IMAGE_NAME"
 
+validate_builds_task:
+    <<: *simple_template
     depends_on:
-        - "gating"
-        - 'container_image_build'
-
-    # VMs created by packer are not cleaned up by cirrus, must allow task to complete
-    auto_cancellation: $CI != "true"
-
-    gce_instance:
-        image_project: "libpod-218412"
-        zone: "us-central1-a"
-        cpu: 4
-        memory: "4Gb"
-        disk: 200
-        image_name: "${IMAGE_BUILDER_CACHE_IMAGE_NAME}"
-        scopes:  # required for image building
-            - compute
-            - devstorage.full_control
-
-    networking_script: '${CIRRUS_WORKING_DIR}/${SCRIPT_BASE}/networking.sh'
-    build_vm_images_script: '$SCRIPT_BASE/build_vm_images.sh |& ${TIMESTAMP}'
-
-    on_failure:
-        failed_df_script: '${DFCMD}'
-        failed_journalctl_b_script: 'journalctl -b || echo "Uh oh, journalctl -b failed"'
-
-
-# Test building of new cache-images for future PR testing, in this PR.
-verify_test_built_images_task:
-
-    only_if: >-
-        $CIRRUS_BRANCH != $DEST_BRANCH &&
-        $CIRRUS_CHANGE_MESSAGE =~ '.*CI:IMG.*' &&
-        $CIRRUS_CHANGE_MESSAGE !=~ '.*CI:DOCS.*'
-
+        - smoke
 
+verify_builds_task:
```

Comment:
note to self: I ***hate** this task name.  This task is for making sure all the externally/manually tested builds (which will be part of the release) are produced.  For example, the current 'build without cgo' and 'build for darwin or windows' items should go here.

This job runs in parallel with integration and system testing, so it's window for including items like 'build each commit' is also large.  If the task gets too long for developer feedback convenience, it can be split up as-needed later on.